scripts/install2

```diff
@@ -101,8 +119,19 @@ if [ -z "$WITH_MESH_POINT" -o "$WITH_MESH_POINT" != "true" -a "$WITH_MESH_POINT"
     else
         echo -e "\e[1;32mMesh Point interface will be configured\e[0m"
         WITH_MESH_POINT=true
+        read -p "Continue with 802.11s instead of older AdHoc (IBSS)?  (Y/n)? " -n 1 -r
```

Comment:
I think we are presenting the terms `Mesh Point`, `802.11s`, and `AdHoc (IBSS)` in a confusing way here.

The question text is also inconsistent with the rest of the questions.